### DISCUSSION TITLE
Strange zoomed appearance on some mobile browsers

### DISCUSSION THREAD

**Discussion by jryans** [community/community#10030] [Other Feature Feedback, Questions, & Ideas]
On at least Firefox for iOS, the search results appear strangely zoomed in and the code samples use a different font size than the line numbers, which pulls them out of alignment:

![IMG_1DB790CE11EE-1](https://user-images.githubusercontent.com/279572/149133912-14b07bca-1066-4fab-8442-b87523daf4f3.jpeg)

**Comment by RockAbeer** [comment]
> On at least Firefox for iOS, the search results appear strangely zoomed in and the code samples use a different font size than the line numbers, which pulls them out of alignment:
> 
> ![IMG_1DB790CE11EE-1](https://user-images.githubusercontent.com/279572/149133912-14b07bca-1066-4fab-8442-b87523daf4f3.jpeg)
